noxfile.py

```diff
@@ -107,7 +91,7 @@ def unit(session):
 
 @nox.session(python=["2.7", "3.5"])
 def unit_prev_versions(session):
-    session.install(*TEST_DEPENDENCIES2)
+    session.install(*TEST_DEPENDENCIES[:-2])
```

Comment:
I am a little nervous about this, expecting to see folks add more deps :D.

can you make two lists one "dependencies" and one "additional_python_3". And for python3 create a set with the two?

Reply:
+1

Reply:
Changed in the latest commit to have 2 lists and then a composition of both for the python3.6+ async tests.